multinet/db.py

```diff
@@ -58,42 +58,41 @@
     port=int(os.environ.get("ARANGO_PORT", "8529")),
     protocol=os.environ.get("ARANGO_PROTOCOL", "http"),
 )
-restricted_keys = {"_rev", "_id"}
 
 
-def db(name: str) -> StandardDatabase:
+def db(name: str, readonly: bool = True, immediate: bool = False) -> StandardDatabase:
     """Return a handle for Arango database `name`."""
-    return arango.db(
-        name,
-        username="root",
-        password=os.environ.get("ARANGO_PASSWORD", "letmein"),
-        verify=True,
+
+    username = "readonly" if readonly else "root"
+    password = (
+        os.environ.get("ARANGO_READONLY_PASSWORD", "letmein")
+        if readonly
+        else os.environ.get("ARANGO_PASSWORD", "letmein")
```

Comment:
You can maybe factor out the if-else expression to apply to just the environment variable.

Reply:
While true, I personally think it would make the code more messy.

Reply:
Ok.